tests/integration/events.bats

```diff
@@ -26,6 +26,34 @@ function teardown() {
 	[[ "${lines[0]}" == *"data"* ]]
 }
 
+@test "events --stats with psi data" {
+	requires root cgroups_v2
+	if [ ! -f /sys/fs/cgroup/cpu.pressure ]; then
+		skip "requires PSI subsystem"
+	fi
```

Comment:
Can you move this to helpers.bash, function `requires`?

Reply:
done, this is now `requires psi`.